SAMtags.tex

```diff
@@ -498,26 +498,33 @@ \subsection{Base modifications}
 This potentially differs to the sequence stored in the main SAM {\sf SEQ} field if the latter has been reverse complemented, in which case SAM {\sf FLAG} 0x10 must be set.
 This means modification positions are also recorded against the original orientation (i.e. starting at the 5' end), and count the original base types.
 
-Each modified base listed also has a quality value associated with it.
+Each modified base prediction listed also has a quality value associated with it.
 Given the unmodified base already has a phred likelihood, this base modification quality should be interpreted as the likelihood of this modification being correct given an assumption the original call is correct.
 
 \begin{description}
-\item[Mm:Z:\tagregex{([ACGTUN][-+]([a-z]+|[0-9]+)(,[0-9]+)*;)*}]
+\item[Mm:Z:\tagregex{([ACGTUN][-+]([a-z]+|[0-9]+)(,[0-9]+)*[.?]?;)*}]
```

Comment:
This regexp is wrong.

Breaking it down, the original was canonical_base `[ACGTUN]`, strand `[+-]`, a list of modified base types or CHEBI ID `([a-z]+|[0-9]+)`, followed by a comma-separated series of deltas: `(,[0-9]+)*` ending in a semicolon.

As far as I understand the intention, the `.` or `?` appear between the strand and base type list, so we'd want:

`([ACGTUN][-+]([a-z]+|[0-9]+)[.?]?(,[0-9]+)*;)*`

Reply:
Thanks for catching this. My intention is for the flag to appear after the base modification code - which I think is what your  proposed regex has? My regex is quite rusty...